dotnet/src/Skills/Skills.Web/Bing/BingConnector.cs

```diff
@@ -32,15 +32,15 @@ public BingConnector(string apiKey, ILogger<BingConnector>? logger = null)
     }
 
     /// <inheritdoc/>
-    public async Task<IEnumerable<string>> SearchAsync(string query, int count = 1, int offset = 0, CancellationToken cancellationToken = default)
+    public async Task<IEnumerable<string>> SearchAsync(string query, string relatedSite = "", int count = 1, int offset = 0, CancellationToken cancellationToken = default)
```

Comment:
Would it make sense to allow passing in an array instead of a single string? This will make it more future proof and often you would like to include multiple (sub)domains. 

```
To see webpages about heart disease from the BBC or CNN websites, type "heart disease" (site:bbc.co.uk OR site:cnn.com).
```

https://support.microsoft.com/en-us/topic/advanced-search-keywords-ea595928-5d63-4a0b-9c6b-0b769865e78a

Reply:
Thank you for your suggestion @iMicknl. 
I agree that passing in an array instead of a single string would be a useful feature for this function.
I will implement this feature in the next commits.